yellowpages/yellowpages-war/src/main/java/org/silverpeas/components/yellowpages/control/YellowpagesSessionController.java

```diff
@@ -103,16 +102,14 @@
   private CompleteContact currentContact = null;
   private String path = null;
   private String owner = "false";
-  // admin || publisher || user
   private String profile;
   private List<YellowPagesGroupDetail> groupPath = new ArrayList<>();
   private boolean portletMode = false;
   private Collection<ContactFatherDetail> currentContacts = null;
   private Collection<UserFull> currentFullUsers = null;
   private Collection<CompleteContact> currentCompleteUsers = null;
-  private String currentTypeSearch;
   private String currentSearchCriteria;
-  public static final String GroupReferentielPrefix = "group_";
+  private static final String GroupReferentielPrefix = "group_";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00115)